community/values/src/main/java/org/neo4j/values/storable/PointValue.java

```diff
@@ -219,7 +220,9 @@ public int computeHash()
     @Override
     public String toString()
     {
-        return format( "Point{%s, %s}", getCoordinateReferenceSystem().getName(), Arrays.toString( coordinate ) );
+        String coordString = coordinate.length == 2 ? format( "x: %s, y: %s", coordinate[0], coordinate[1] ) :
```

Comment:
I think it would have been better to separate toString() from toPrettyString(), but this is good enough for now, and we can make a cleaned separation of internal vs. external display later.